roles/ideascube/tasks/branding.yml

```diff
@@ -0,0 +1,7 @@
+---
+- include_vars: group_vars/{{ ansible_architecture }}.yml
+  tags: ['master', 'custom']
+
```

Comment:
Hmm, but the branding is done before installing the package. I guess the folder does need to be created then, doing it now.